game/scripts/npc/items/item_bfury.txt

```diff
@@ -13,7 +13,7 @@
     "AbilityTextureName"                                  "custom/recipe/recipe_1"
     "ItemRequirements" //OAA
     {
-      "01"                                                "item_broadsword;item_claymore;item_pers;item_quelling_blade"
+      "01"                                                "item_broadsword;item_claymore;item_void_stone;item_quelling_blade"
```

Comment:
Instead of removing the stat that you hate, you can just remove Claymore from the recipe. Sounds like a better option to me.